README.md

```diff
@@ -1,24 +1,21 @@
 # Cerbos Python Demo
 
-<img src="https://github.com/cerbos/cerbos/blob/main/docs/supplemental-ui/logo.png?raw=true" alt="Cerbos" width="87" height="100"/>
+This project demonstrates how to work with [Cerbos](https://github.com/cerbos/cerbos) in Python projects.
 
-This project `demo-python` will help you get familiar with using [Cerbos](https://github.com/cerbos/cerbos) in your Python applications.
-
-It's an app that follows access control regulation of a company employee `Harry` in process from submitting it, to getting his holiday request approved. Simply put, `demo-python` shows what are `Harry` and his work colleagues allowed to do with his `holiday request`.
+It shows how Cerbos enforces access rules on a holiday request created by an employee named Harry.
 
 ## Requirements
 - Minimum Python version supported by this demo is 3.10
-- Your Cerbos server is up and running (via either a [binary](https://docs.cerbos.dev/cerbos/latest/installation/binary.html), [container](https://docs.cerbos.dev/cerbos/latest/installation/container.html) or [helm](https://docs.cerbos.dev/cerbos/latest/installation/helm.html))
+- Your Cerbos server is up and running (check how to run a [binary](https://docs.cerbos.dev/cerbos/latest/installation/binary.html), [container](https://docs.cerbos.dev/cerbos/latest/installation/container.html) or [Helm chart](https://docs.cerbos.dev/cerbos/latest/installation/helm.html))
```

Comment:
The demo starts Cerbos automatically so it's not necessary to have it installed.

Reply:
Nice! I got misled by my recent Docker issue into thinking that it had to be manually triggered..